docs/operator_parameters.md

```diff
@@ -0,0 +1,299 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+*CONFIG_MAP_NAME* environment variable. Variable names are underscore-separated
+words.
+
+Available Configuration Parameters
+----------------------------------
+
+### General
+* **etcd_host**
+  Etcd *host:port* for Patroni. Not required when Patroni native Kubernetes
+  support is used. The default is empty (use Kubernetes-native DCS).
+
+* **docker_image**
+  Spilo docker image for Postgres instances. For production, don't rely on
+  defaults, as it might be too old. Instead, build your own Spilo image from
+  the [github repository](github.com/zalando/spilo).
```

Comment:
Is it a correct reference? Looks like without 'https' github will add 'https://github.com/zalando-incubator/postgres-operator/docs' before this path.